content/updates/governance/1-More-powerful-associations.md

```diff
@@ -0,0 +1,33 @@
+# More powerful product associations
+
+## New association type with quantities
+::: meta-data type="New" ee-only="false" link-to-doc="../articles/manage-your-association-types.html#create-an-association-type-with-quantities"
+:::
+
+The new association type with quantities enables you to create product sets. You can associate several products together and define quantities for each of them. 
+
+For instance, you can easily create a dining table set composed of 1 table and 4 chairs, or a bedding set with 1 duvet cover and 2 pillowcases.
+
+Once you have created the right association type, and associated your products in the PIM, it will look like this:
+
+![This is a product set composed of different associated products with quantities in the PIM](../img/Dining-set_Felix_PIM.png)
+
+## New two-way association type
+::: meta-data type="New" ee-only="false" link-to-doc="../articles/manage-your-association-types.html#create-a-2-way-association-type"
+:::
+
+Easily handle product compatibility in your PIM thanks to our brand new two-way association type! If you choose this new association type, and associate product A with product B, product B will automatically be associated with product A.
+
+Use a two-way association if you need to easily link substitute or replacement items, or products that are compatible such as appliances and spare parts.
+
+## Small UI improvements
+::: meta-data type="Improvement" ee-only="false"
+:::
+
+We made some small improvements to make your life easier when working with product associations:
+- We now display the total number of associations per product or product model nearby the `Associations` tab when you are on your product form. You can now see the number of associations for your products at a glance.
+- We also show you the total number of associations per association type, when you are on your product associations page.
+![Associations number](../img/associations-number.png)
+- From the `Associations` tab, you can now click on an associated product or product model to open it in a new tab.
```

Comment:
From the product edit form, in the `Associations` tab, you can now click on an associated product or product model to open it in a new tab.